modules/javafx.web/src/test/java/test/javafx/scene/web/LocalStorageTest.java

```diff
@@ -40,19 +41,16 @@
 
 public class LocalStorageTest extends TestBase {
 
-    private static final File LOCAL_STORAGE_DIR = new File("LocalStorageDir");
+    private static final File LOCAL_STORAGE_DIR_PATH = new File("build/");
+    private static final File LOCAL_STORAGE_DIR = new File("build/localstorage");
```

Comment:
I don't see the need for two variables here. Why not use the updated value of `LOCAL_STORAGE_DIR` everywhere?